projects/igniteui-angular/src/lib/core/styles/components/list/_list-theme.scss

```diff
@@ -373,7 +373,7 @@
     }
 
     %igx-list__item-lines {
-        color: --var($theme, 'item-text-color');
```

Comment:
I merged the change from master to 8.1.x. This line is still changed in 8.1.x though. Should it stay like this?

Reply:
@desig9stein ?